setup.py

```diff
@@ -125,7 +125,8 @@ def getsitepackages():
                                      "python" + sys.version[:3],
                                      "site-packages"),
                         os.path.join(prefix, "lib", "site-python"),
-                        os.path.join(prefix, "python" + sys.version[:3], "lib-dynload")]
+                        #os.path.join(prefix, "python" + sys.version[:3], "lib-dynload")
+                        ]
```

Comment:
I think it's fine. Now that 0.8.1 is out let's remove it and see if anything breaks.